specs/commandline/verify.md

```diff
@@ -40,61 +52,82 @@ An example of `trustpolicy.json`:
     "trustPolicies": [
         {
             // Policy for all artifacts, from any registry location.
-            "name": "wabbit-networks-images",  // Name of the policy.
-            "registryScopes": [ "*" ],         // The registry artifacts to which the policy applies.
-            "signatureVerification": {         // The level of verification - strict, permissive, audit, skip.
+            "name": "wabbit-networks-images",                         // Name of the policy.
+            "registryScopes": [ "localhost:5000/net-monitor" ],       // The registry artifacts to which the policy applies.
+            "signatureVerification": {                                // The level of verification - strict, permissive, audit, skip.
                 "level": "strict"
             },
-            "trustStores": [ "ca:wabbit-networks" ], // The trust stores that contains the X.509 trusted roots.
-            "trustedIdentities": [                   // Identities that are trusted to sign the artifact.
+            "trustStores": [ "ca:wabbit-networks" ],                  // The trust stores that contains the X.509 trusted roots.
+            "trustedIdentities": [                                    // Identities that are trusted to sign the artifact.
                 "x509.subject: C=US, ST=WA, L=Seattle, O=wabbit-networks.io, OU=Finance, CN=SecureBuilder"
             ]
         }
     ]
 }
 ```
```

Comment:
I will like to keep an example for "*" in the registry scope. So instead of replacing the above example, could we include two examples, one with "*" in registry scope, and another with a specific repository?